### PR TITLE
FreeBSD 10 and higher use Fortuna, not Yarrow

### PR DESCRIPTION
**Pull request by ashemedai**
See https://www.freebsd.org/cgi/man.cgi?query=random&apropos=0&sektion=4&manpath=FreeBSD+12-current&arch=default&format=html#end for background info.

### CODE REVIEW

File: docs/hazmat/backends/openssl.rst [hunk 1]
```diff
@@ -93,7 +93,8 @@ OS random sources
 -----------------
 
 On macOS and FreeBSD ``/dev/urandom`` is an alias for ``/dev/random`` and
-utilizes the `Yarrow`_ algorithm.
+utilizes the `Yarrow`_ algorithm. FreeBSD version 10 and higher use the
+`Fortuna`_ algorithm.
```
Comment (alex) [inline docs/hazmat/backends/openssl.rst]: According to our docs, we only support FreeBSD 11 and up, so let's simplify this to saying macOS is Yarrow and FreeBSD is Fortuna

**Comment by alex** [comment]
One small note, thanks for the PR!

**Comment by alex** [comment]
@ashemedai you might need to push an empty commit or something; Travis seems confused.

**Comment by ashemedai** [comment]
Jenkins error caused by: https://github.com/pypa/setuptools/issues/1042

**Comment by alex** [comment]
Coverage results are manifestly impossible. Docs only so I'm merging.

**Pull request merged**